caravel/helpers.py

```diff
@@ -120,4 +126,76 @@ def _version_text(sep):
     from _version import __version__ as caravel_version
     from looper import __version__ as looper_version
     return "caravel version: {}".format(caravel_version) + sep + \
-           "looper version: {}".format(looper_version)
+           "looper version: {}\n".format(looper_version)
+
+
+def print_terminal_width(txt=None, char="-"):
+    """
+    Print a line composed of the chars and a text in the middle of the terminal
+
+    :param str txt: a string to display in the middle of the terminal
+    :param str char: a character that the box will be composed of
+    :return None
+    """
+    char = str(char)
+    assert len(char) == 1, "The length of the char parameter has to be equal 1, got '{}'".format(len(char))
+    spaced_txt = txt.center(len(txt)+2) if txt is not None else ""
+    print(char * int((terminal_width() / 2) - (len(spaced_txt) / 2)) + spaced_txt + char *
+          int((terminal_width() / 2) - (len(spaced_txt) / 2)))
+
+
+def terminal_width():
+    """
+    Get terminal width
+
+    :return: width of the terminal
+    :rtype int
```

Comment:
Return structure should be `:return <type>:`

Reply:
I followed sphinx style docstrings here: http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists

Which format do we generally follow?

Reply:
TLDR: we can autogenerate API docs if we inline, and it matches what we typically do for the `param` ones.

Fuller story:
Sorry, yeah it's really muddled, and we've been inconsistent before. We have some both in this style (separate `rtype`) and some inline. I actually prefer Numpy style, but our `MkDocs` theme will parse the inline sphinx style to render the API docs from these docstrings, and i think inline looks a bit better since we're already doing it for `param`.
So I guess i don't really have a strong intrinsic preference, but pragmatically we get the autodoc benefit with inline style, and more of our current code uses inline, and I think those are nice wins.

Reply:
sure, I don't have a preference either. Autogenerating docs is a good enough reason to change to inline, then